files/zh-cn/web/javascript/guide/using_promises/index.md

```diff
@@ -341,7 +341,7 @@ console.log(1); // 1, 2, 3, 4
 
 简便的 Promise 链式编程最好保持扁平化，不要嵌套 Promise，因为嵌套经常会是粗心导致的。可查阅下一节的[常见错误](#常见错误)中的例子。
 
-嵌套 Promise 是一种可以限制 `catch` 语句的作用域的控制结构写法。明确来说，嵌套的 `catch` 仅捕捉在其之前同时还必须是其作用域的 failureres，而捕捉不到在其链式以外或者其嵌套域以外的 error。如果使用正确，那么可以实现高精度的错误修复。
+嵌套 Promise 是一种可以限制 `catch` 语句的作用域的控制结构写法。明确来说，嵌套的 `catch` 仅捕捉在其之前同时还必须是其作用域的 failures，而捕捉不到在其链式以外或者其嵌套域以外的 error。如果使用正确，那么可以实现高精度的错误修复。
```

Comment:
Please translate the terms into Chinese rather than keep it in the localized document.

```suggestion
嵌套 Promise 是一种可以限制 `catch` 语句的作用域的控制结构写法。明确来说，嵌套的 `catch` 仅捕捉在其之前同时还必须是其作用域的错误，而捕捉不到在其链以外或者其嵌套域以外的错误。如果使用正确，那么可以实现高精度的错误修复。
```